Cleanup createStructuredDirectory

Hwesta opened the pull request:
Move createStructuredDirectory into archivematicaFunctions and rename create_structured_directory so it can take advantage of the directory constants and functions already there.  Add manual normalization constants to that file as well.  Update all callers, remove now deprecated archivematicaCreateStructuredDirectory.[py|sh], and do some minor cleanup.

Related to AIP Reingest.

mistydemeo commented on src/MCPClient/lib/clientScripts/createSIPfromTransferObjects.py:
```diff
@@ -47,11 +42,11 @@
     #create row in SIPs table if one doesn't already exist
     lookup_path = destSIPDir.replace(sharedPath, '%sharedPath%')
-    sql = """SELECT sipUUID FROM SIPs WHERE currentPath = '""" + MySQLdb.escape_string(lookup_path) + "';"
+    sql = """SELECT sipUUID FROM SIPs WHERE currentPath = '""" + MySQLdb.escape_string(lookup_path) + """';"""
```
Update these to use the SQL placeholders introduced in #110 instead of worrying about escaping?

mistydemeo commented:
:thumbsup:

Hwesta committed:
Update createStructuredDirectory

Move createStructuredDirectory into archivematicaFunctions and rename
create_structured_directory so it can take advantage of the directory
constants and functions already there.  Add manual normalization
constants to that file as well.  Update all callers, remove now
deprecated archivematicaCreateStructuredDirectory.py/sh, and do some
minor cleanup.

Hwesta commented:
Updated SQL in both client scripts touched to use fixes from #110

mistydemeo commented:
Still :+1:

Hwesta commented:
Removed commit that added the SQL 'fixes' since they're unnecessary with #98
